openslides/users/static/js/users/site.js

```diff
@@ -609,60 +613,58 @@ angular.module('OpenSlidesApp.users.site', [
                 });
             }
         };
-        // delete all selected users
-        $scope.deleteMultiple = function () {
-            angular.forEach($scope.users, function (user) {
+        var selectModeAction = function (predicate) {
+            angular.forEach($scope.usersFiltered, function (user) {
                 if (user.selected) {
-                    User.destroy(user.id);
+                    predicate(user);
                 }
             });
             $scope.isSelectMode = false;
             $scope.uncheckAll();
         };
-        // delete single user
-        $scope.delete = function (user) {
-            User.destroy(user.id);
+        // delete all selected users
+        $scope.deleteMultiple = function () {
+            selectModeAction(function (user) {
+                $scope.delete(user);
+            });
         };
         // add group for selected users
         $scope.addGroupMultiple = function (group) {
-            angular.forEach($scope.users, function (user) {
-                if (user.selected) {
-                    user.groups_id.push(group);
-                    User.save(user);
-                }
+            if (!group) return;
```

Comment:
Please add `{ }` and please use positive condition check: if group then call selectModeAction ...